src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/TraceLogging/XplatEventLogger.cs

```diff
@@ -160,12 +271,6 @@ private static void AppendByteArrayAsHexString(StringBuilder builder, byte[] byt
 
         protected internal override void OnEventSourceCreated(EventSource eventSource)
         {
-            // Don't enable forwarding of NativeRuntimeEventSource events.`
-            if (eventSource.GetType() == typeof(NativeRuntimeEventSource))
```

Comment:
If I recall correctly, this is here because otherwise events that are written by the runtime end up getting pushed to this `EventListener` and then back down to LTTng.  This only happens when COMPlus_EnableEventLog=1, and then you only see it if you're capturing via LTTng I believe.  The details are a bit fuzzy...

Reply:
This was preventing the provider to get enabled when only LTTng is specified via COMPlus_EnableEventLog=1. I moved this check to `OnEvenWritten` check instead so that they don't get forwarded as EventSourceEvent.